sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLPoolDAO.java

```diff
@@ -849,7 +849,7 @@ public Pool mapRow(ResultSet rs, int rowNum) throws SQLException {
         }
 
         if (!isLazy()) {
-          p.setExperiments(experimentDAO.listByPoolId(id));
+          // BATS Hibernate will load this
```

Comment:
`Study_Experiment` is empt.  I guess if nothing is referencing it then it is fine to rem.